recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "0.4.5" %}
+{% set version = "0.4.6" %}
```

Comment:
@grlee77 

We changed the testing engine from `nose` to `pytest` so line 33 below needs to require `pytest` instead of `nose`.

Also, since I'm the new maintainer of the scikit-sparse package can I be made a maintainer on this recipe?

Reply:
thanks, I updated the recipe to use pytest